androidApp/src/main/java/com/example/tasky/android/agenda/components/DetailsPhotoSection.kt

```diff
@@ -0,0 +1,139 @@
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Add
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme.typography
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil3.compose.AsyncImage
+import com.example.tasky.android.R
+import com.example.tasky.android.theme.Black
+import com.example.tasky.android.theme.Gray
+import com.example.tasky.android.theme.Light2
+import com.example.tasky.android.theme.LightBlue
+import com.example.tasky.android.theme.MyApplicationTheme
+import com.example.tasky.model.agenda.Photo
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
+import kotlinx.collections.immutable.toImmutableList
+
+@Composable
+fun DetailsPhotoSection(
+    photos: ImmutableList<Photo>,
+    isEdit: Boolean,
+    onAddClick: () -> Unit,
+    onPhotoClick: (Photo) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Box(modifier = modifier.background(Light2)) {
+        if (photos.isEmpty()) {
+            Row(
+                modifier =
+                    Modifier
+                        .align(Alignment.Center)
+                        .clickable(onClick = onAddClick),
+            ) {
+                Icon(Icons.Outlined.Add, contentDescription = null, tint = Gray)
+                Spacer(modifier = Modifier.width(16.dp))
+                Text(stringResource(R.string.add_photos), style = typography.bodyMedium, lineHeight = 18.sp, color = Gray)
+            }
+        } else {
+            Column(modifier = Modifier.padding(16.dp)) {
+                Text(stringResource(R.string.photos), style = typography.headlineMedium, lineHeight = 18.sp, color = Black)
+                Spacer(modifier = Modifier.height(16.dp))
+                LazyRow(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+                    items(photos) {
```

Comment:
Here it would be really easy to give these photos unique keys in this LazyRow to only have to update those composables that changed with a state update. Also, you can then easily add animations to list items with the `animateItem()` modifier